jorge/css/style.css

```diff
@@ -0,0 +1,30 @@
+.nav_container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  background-color: blue;
+  padding: 32px 64px;
+}
+
+.nav_logo {
+  width: 250px;
+}
+
+.nav_buttons {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  background-color: white;
+  padding: 16px;
+}
+
+.nav_btn {
```

Comment:
This name is confusing when you also have `nav_buttons`, if this class is meant to be generic (applied as a common set of rules to all buttons in the page) then create a `btn` class on its own